pkgs/tools/backup/duplicity/default.nix

```diff
@@ -11,40 +11,48 @@
 , rsync
 , makeWrapper
 , gettext
+, getconf
+, testers
+, nix-update-script
 }:
 
-python3.pkgs.buildPythonApplication rec {
+let self = python3.pkgs.buildPythonApplication rec {
   pname = "duplicity";
-  version = "0.8.23";
+  version = "2.2.3";
 
```

Comment:
Looks like it supports `pyproject = true` now:

https://gitlab.com/duplicity/duplicity/-/compare/rel.0.8.23...rel.2.2.3?page=9#5d07e7d72637aa0d59c89d381fe6dc4cf46e2491